.env.sample

```diff
@@ -1,3 +1,13 @@
+# used by the server e.g. to create tokens
+SECRET_KEY_BASE=super-secret-key
+
+# true, if WebRTC peers are used
+WEBRTC_USED=true
+
+# hostname used to generate URLs throught the server
 VIRTUAL_HOST=localhost
+
+# TURN default configuration
 INTEGRATED_TURN_IP=127.0.0.1
```

Comment:
What do you think about adding information that loopback should be used only in local development?